exploring_UV_extinction_curves/Exploring_UV_extinction_curves.ipynb

```diff
@@ -0,0 +1,1662 @@
+{
```

Comment:
<p>We will use the <code>changeNaN</code> function to locate and change these problematic values.</p><p> </p><p>I would demo one file with the NaNs to motivate this function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>